style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+}
```

Comment:
kindly consider removing a general font-size from your document 😄 since the content is a mashup of several fonts.